hack/generator/pkg/astmodel/package_definition.go

```diff
@@ -175,7 +175,7 @@ func emitGroupVersionFile(pkgDef *PackageDefinition, outputDir string) error {
 
 	gvFile := filepath.Join(outputDir, "groupversion_info"+CodeGeneratedFileSuffix)
 
-	err = ioutil.WriteFile(gvFile, buf.Bytes(), 0700)
+	err = ioutil.WriteFile(gvFile, buf.Bytes(), 0600)
```

Comment:
What does this magic number mean?

Reply:
`rw` for owner, nothing for anyone else (previously it was `rwx`, which `gosec` objects to)

Reply:
obviously having constants defined in the stdlib is too ivory-tower 😉